packages/field-base/src/clear-button-mixin.js

```diff
@@ -51,6 +51,8 @@ export const ClearButtonMixin = (superclass) =>
       super.ready();
 
       if (this.clearElement) {
+        this.clearElement.addEventListener('mousedown', (event) => this._onClearButtonAction(event));
+        this.clearElement.addEventListener('touchstart', (event) => this._onClearButtonAction(event, false));
         this.clearElement.addEventListener('click', (event) => this._onClearButtonClick(event));
```

Comment:
Keeping the `click` listener to keep the behavior aligned with Combo Box:

https://github.com/vaadin/web-components/blob/7cc4252e925d318d49aacf00b2e8878f1641299e/packages/combo-box/src/vaadin-combo-box-mixin.js#L626-L633

`_isClearButton` uses the composed path to check the origin of the click.